### PR TITLE
Update 1_0_20_duckiebot_SD_initialization.md

### PR DESCRIPTION
**Pull request by phideltaee**
updated rh1 initialization procedure for DB-beta - ethz 2020 amod class and replaced references to raspi with general references for both raspi and JN.

### CODE REVIEW

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ A valid `hostname` satisfies all the following requirements:
 
 Warning: this currently only works on Ubuntu. Mac is not supported.
 
-Warning: on Ubuntu 16, you need to remove and re-insert the SD card. On Ubuntu 18 this is not necessary.
+Warning: on Ubuntu 16, you need to remove and re-insert the SD card. On Ubuntu 18 or 20 this is not necessary.
```
Comment (afdaniele) [inline book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:33]: use "Ubuntu 18 and above" instead.

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md [hunk 2]
```diff
@@ -101,12 +105,22 @@ sr0     11:0    1  1024M  0 rom
...
+
 - You will then have to enter your laptop's `sudo` password to run Etcher.
 
 - When asked "Are you sure?" select <kbd>y</kbd>.
 
 Warning: Always be careful when selecting disk to be imaged. You don't want to lose your computer's system partition!
 
+For Jetson Nano:
+
+- The procedure will ask to accept the conditions for use. When asked "Do you accept? (a=Accept, n=Reject, r=Read License) \[n]: " select <kbd>a</kbd>.
```
Comment (afdaniele) [inline book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:118]: Add suggestion to press `r` and read the license before accepting it.

**Review by afdaniele — CHANGES_REQUESTED** [review]

**Comment by afdaniele** [comment]
Nice! Merging!

**Review by afdaniele — APPROVED** [review]

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Duckiebot SD Card Initialization {#setup-duckiebot status=ready}
 
-This page is for the `DB18` configuration and above. 
+This page is for the `DB18` configuration and above (including Jetson Nano configurations). 
 
 <div class='requirements' markdown="1">
 
-Requires: An SD card of size at least 16 GB.
+Requires: An SD card of size at least 16 GB. 32 GB preferred.
```
Comment (AleksandarPetrov) [inline book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:7]: 32GB is *required* for Jetson, 16 won't work.

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md [hunk 4]
```diff
@@ -49,6 +49,8 @@ The basic options are:
                        the flashed device The default is: quackquack
```
Comment (AleksandarPetrov) [inline book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:49]: We need to remove the username and password from here (and, @afdaniele also from dts itself, right?). And we need to mention that they will be the default.

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md [hunk 5]
```diff
@@ -49,6 +49,8 @@ The basic options are:
...
     --wifi             default: duckietown:quackquack
     --country          default: US
+    --type             The type of your device. E.g. `duckiebot`.
+    --configuration    The configuration of your robot. This is associated with `--type` option. E.g. `DB-beta`.
```
Comment (AleksandarPetrov) [inline book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:53]: for type and configuration maybe add all the (important) options?

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md (hunk 3 above)
Comment (phideltaee) [reply book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:7]: will a larger sd work as well?

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md (hunk 4 above)
Comment (phideltaee) [reply book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:49]: I can just move them to the non-essential commands. Or, are they not going to be valid commands anymore?

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md (hunk 5 above)
Comment (phideltaee) [reply book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md:53]: included watchtower, traffic_light and duckiebot for types. For configurations DB-beta, DB20, DB19, and DB18.

**Pull request merged**